### DISCUSSION TITLE
VenaChain RPC接口配置HTTPS时，需要做跨域处理设置

### DISCUSSION THREAD

**Discussion by 99Kies** [NonceGeek/Tai-Shang-Soul-Card#60] [General]
修改`release/linux/scripts/start-node.sh`文件。
添加一个 options (--rpcvhosts "*")
![image](https://user-images.githubusercontent.com/45918704/187827039-9cf2d0c4-043c-4799-8a99-e48908b775ae.png)

**Comment by leeduckgo** [comment]
Good Job.
